CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+Package Contribution Guidelines
+===============================
+
+If you are submitting a new Spack package, please observe the
+following standards:
+
+1. Code must be [PEP-8 compliant](http://spack.readthedocs.io/en/latest/contribution_guide.html?highlight=pep-8#flake8-tests)
+
+2. At least one numeric (checksummable) version must be supplied.  If
+   this package does not yet have any official releases, create an
+   unofficial checksummable Spack-only release using something like:
+   ```
+   version('1.9.5.1.1',
+        git='https://github.com/jswhit/pyproj.git',
+        commit='0be612cc9f972e38b50a90c946a9b353e2ab140f')
+   ```
+
+2. All explicit versions in `depends_on()` statements must be for
```

Comment:
I have no problem with packages that do this:

```python
    depends_on('foo@develop', when='develop')
```

I think this is something we should add automated checks for.  Homebrew has a `brew audit` command that does this kind of stuff for its formulae.